frontend/app/src/components/dashboard/LiquidityProviderBalanceTable.vue

```diff
@@ -213,4 +250,39 @@ const percentageOfTotalNetValue = (value: BigNumber) => {
 const percentageOfCurrentGroup = (value: BigNumber) => {
   return calculatePercentage(value, get(totalInUsd));
 };
+
+const premium = getPremium();
+
+const fetch = (refresh: boolean = false) => {
+  fetchUniswapV3Balances(refresh);
+  fetchUniswapV2Balances(refresh);
+
+  if (get(premium)) {
+    fetchSushiswapBalances(refresh);
+    fetchBalancerBalances(refresh);
+  }
+};
+
+onBeforeMount(() => {
+  console.log('test');
```

Comment:
Interesting there was no warning/error for this?

Reply:
weird the linter didn't catch it